### ISSUE TITLE
Windows DLL available to download

### ISSUE THREAD

**Issue by macintoshplus** [php/pecl-processing-rrd#5]
I am pleased to announce that this extension is now compiled for Windows by the PHP Extension Repository site.

You can download the DLLs on the extension page: https://phpext.phptools.online/extension/processing/rrd-316

If you have any questions, I am available to discuss them.

**Comment by MirKml** [comment]
seems it's closed, migrated to another platform ...

**Comment by macintoshplus** [comment]
Yes, the site is closed for some reason: https://nahan.fr/la-fin-de-php-extension-repository

The main way is to make this extension compatible with PIE. https://github.com/php/pie
